src/mumble/TalkingUIEntry.h

```diff
@@ -57,6 +57,8 @@ class TalkingUIEntry : public TalkingUIComponent {
 
 	virtual void setDisplayString(const QString &displayString) = 0;
 
+	virtual void setBackgroundColor(const QString &color) = 0;
```

Comment:
Instead of introducing a new function here, you could also make use of the `getStylableWidget()` member function to access the contained `MultiStyleWidgetWrapper` and set the bg color on that directly.